javascript/javascript1/week2/eventApp.js

```diff
@@ -0,0 +1,29 @@
+//Event Application Function
+function getEventWeekday(daysFromToday) {
+  const dateToday = new Date(); //to get the exact date and time
+  const dayNumber = dateToday.getDay(); //returns a number of days from 0-6 (starts from sunday)
+
+  const nameOfDays = [
+    "Sunday",
+    "Monday",
+    "Tuesday",
+    "Wednesday",
+    "Thursday",
+    "Friday",
+    "Saturday",
+  ];
+
+  const indexes = daysFromToday + dayNumber;
+  if (indexes >= 7) {
+    const r = indexes % 7; //since nameOfDays has only 7 elements need to find the remainder
+    return nameOfDays[r];
+  } else {
+    return nameOfDays[indexes];
+  }
+}
+
+// With todays weekday a friday
+console.log(getEventWeekday(8)); // Logs out "Saturday"
```

Comment:
I'd recommend you clear out the comments with `// Logs out "Saturday"` because if I don't review the code on the right day, then it will show a different result.

Reply:
Logs out comments removed in event application...